From 2e74326d526e256077b30c1395f2e8bdd96ac2c0 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Thu, 23 Jan 2014 18:39:23 -0800
Subject: ASoC: rsnd: remove pin sync option

Renesas Chip is supporting multi pin sound,
but the HW setting is very difficult and confusable.
But driver is supporting it halfway.
Remove SYNC option at this point.

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit 6f3ab6c1c022e7a4877d38940cd45ae7804a15e2)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 include/sound/rcar_snd.h | 1 +
 sound/soc/sh/rcar/ssi.c  | 2 --
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/sound/rcar_snd.h b/include/sound/rcar_snd.h
index 6add6ccc811e..1d19bfc2486d 100644
--- a/include/sound/rcar_snd.h
+++ b/include/sound/rcar_snd.h
@@ -35,6 +35,7 @@
  */
 #define RSND_SSI_CLK_PIN_SHARE		(1 << 31)
 #define RSND_SSI_SYNC			(1 << 29) /* SSI34_sync etc */
+#define RSND_SSI_CLK_FROM_ADG		(1 << 30) /* clock parent is master */
 
 #define RSND_SSI_PLAY			(1 << 24)
 
diff --git a/sound/soc/sh/rcar/ssi.c b/sound/soc/sh/rcar/ssi.c
index ddcca067908c..68393a9f91bf 100644
--- a/sound/soc/sh/rcar/ssi.c
+++ b/sound/soc/sh/rcar/ssi.c
@@ -136,8 +136,6 @@ static void rsnd_ssi_mode_set(struct rsnd_priv *priv,
 			val = adg;			\
 		else					\
 			val = ext;			\
-		if (flags & RSND_SSI_SYNC)		\
-			val |= sync;			\
 	} while (0)
 
 	flags = rsnd_ssi_mode_flags(ssi);
-- 
2.1.2

